routers/repo/view.go

```diff
@@ -36,6 +36,45 @@ const (
 	tplMigrating base.TplName = "repo/migrating"
 )
 
+// FIXME: There has to be a more efficient way of doing this
+func getReadmeFileFromPath(commit *git.Commit, treePath string) (*git.Blob, error) {
+	tree, err := commit.SubTree(treePath)
+	if err != nil {
+		return nil, err
+	}
+
+	entries, err := tree.ListEntries()
+	if err != nil {
+		return nil, err
+	}
+	entries.CustomSort(base.NaturalSortLess)
```

Comment:
I'm not entirely sure why this sort is needed here. 🤔

Reply:
Predictable results. "readme" "Readme" "ReAdME" and "README" etc are all readme files. We want the same one picked always. We want the same one picked if you look in docs directly or not.